firefox_puppeteer/ui/toolbars.py

```diff
@@ -446,3 +446,21 @@ def verifier(self):
         :returns: Reference to the identity-popup content verifier.
         """
         return self.marionette.find_element(By.ID, 'identity-popup-content-verifier')
+
+    def close(self, force=False):
+        """Closes the identity popup.
+
+        :param force: If true, the popup is closed by its own hide function,
+                      otherwise a key event is sent to close the popup.
+        """
+        if not self.is_open:
+            return
+
+        if force:
+            self.marionette.execute_script(
+                "arguments[0].hidePopup();",
+                script_args=[self.popup])
```

Comment:
Please also check the prefs library for the correct formatting of code for `execute_script`.